subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -777,6 +777,7 @@ static void isr_abort(void *param)
 static void isr_done_cleanup(void *param)
 {
 	if (lll_is_done(param)) {
+		radio_status_reset();
 		return;
 	}
```

Comment:
Good catch, move the reset outside the return as generally the rest of the cleanup code path is missing the reset even when `lll_is_done` is false. You may send this commit as separate PR for early approval to merge.

```suggestion
{
	/* Clear radio status and events */
	lll_isr_status_reset();
	
	if (lll_is_done(param)) {
		return;
	}
```